tests/unit/functions/test_expression_evaluator.py

```diff
@@ -7,493 +7,16 @@
     StateIterationItemNotSetError,
 )
 from data_factory_testing_framework.exceptions.variable_not_found_error import VariableNotFoundError
-from data_factory_testing_framework.functions.expression_evaluator import ExpressionEvaluator
+from data_factory_testing_framework.functions import ExpressionEvaluator
 from data_factory_testing_framework.pythonnet.csharp_datetime import CSharpDateTime
 from data_factory_testing_framework.state.dependency_condition import DependencyCondition
 from data_factory_testing_framework.state.pipeline_run_state import PipelineRunState
 from data_factory_testing_framework.state.pipeline_run_variable import PipelineRunVariable
 from data_factory_testing_framework.state.run_parameter import RunParameter
 from data_factory_testing_framework.state.run_parameter_type import RunParameterType
-from lark import Token, Tree
 from pytest import param as p
 
 
-@pytest.mark.parametrize(
-    ["expression", "expected"],
-    [
-        p("value", Tree(Token("RULE", "literal_evaluation"), [Token("LITERAL_LETTER", "value")]), id="string_literal"),
-        p(
-            " value ",
-            Tree(Token("RULE", "literal_evaluation"), [Token("LITERAL_LETTER", "value")]),
-            id="string_with_ws_literal",
-            marks=pytest.mark.skip(""),
-        ),
-        p("11", Tree(Token("RULE", "literal_evaluation"), [Token("LITERAL_INT", "11")]), id="integer_literal"),
-        p(
-            "@pipeline().parameters.parameter",
```

Comment:
Any reason on removing all these tests that validate whether an expression is resulting in the correct tree?